content/blog.md

```diff
@@ -0,0 +1,63 @@
+## Blog
```

Comment:
Why move from marketing to content? The department is marketing, their objectives are met through content?

Reply:
Yep, fair, will change back